infra/metadata.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       configurationSecs: 60
       deploymentSecs: 300
     costEstimate:
-      description: $140 per month
+      description: "cost of this solution is $140"
```

Comment:
**Thought:**
I wonder what the purpose of this `costEstimate` field is, and if the existing wording `$140 per month` was intentional.
I know this `metadata.yaml` will eventually be used to populate parts of [Solutions-in-Console](https://console.cloud.google.com/products/solutions/catalog), but I'm not sure if that's also the plan with this `costEstimate` field.
No action needed.